homeassistant/helpers/config_validation.py

```diff
@@ -219,6 +221,17 @@ def validate(obj):
     return validate
 
 
+def validate_values(valid_values):
```

Comment:
Calling `keys()` on a dict will not return a list but an iterable `dict_keys` object. However, it should still work with `vol.In`